conference.js

```diff
@@ -1676,6 +1676,18 @@ export default {
                 });
             }
         );
+
+        room.on(
+            JitsiConferenceEvents.SILENT_STATUS_CHANGED,
+            (id, isSilent) => {
```

Comment:
Why do we need it and here, isn't the one in `base/conference/actions.any.ts` enough?

Reply:
I thought the same thing, but if I remove the listener from `conference.js`, other browser-based participants don't see the silent status. And if I remove the listener in `base/conference/actions.any.ts`, mobile app participants don't see the status. I could only get it to work on both platforms with both of these listeners. I never could figure out why that's the case though.